scripts/python_api_client.py

```
@@ -1,80 +1,407 @@
 
-    def check_rebalance_status(self, portfolio_id: str) -> Dict[str, Any]:
-        """Check the status of an ongoing or recent rebalance."""
-        response = self.session.get(f"{self.base_url}/portfolio/{portfolio_id}/rebalance-status")
-        response.raise_for_status()
-        return response.json()
+    def get_rebalance_summary(self, portfolio_id: str) -> Dict[str, Any]:
+        """Fetch the rebalance readiness summary for a portfolio."""
+        return self._get(f"/rebalance/summary/{portfolio_id}")
+
+    def export_portfolio(
+        self, portfolio_id: str, format: str = "json"
```

Ruff (0.15.18)

[error] Function argument `format` is shadowing a Python builtin

(A002)